core/context.go

```diff
@@ -24,8 +23,6 @@ type Context struct {
 	// Frame receives from client.
 	Frame frame.Frame
 
-	// Metadata is be built from handshake.
-	Metadata metadata.Metadata
```

Comment:
Metadata has been moved to the data stream. The data stream now holds metadata that has been merged from both the connection-level and stream-level metadata.